### DISCUSSION TITLE
Full screen mode while viewing code in landscape mode

### DISCUSSION THREAD

**Discussion by swushi** [community/community#3467] [Mobile]
![image](https://user-images.githubusercontent.com/34108400/109695049-159b5180-7b51-11eb-8eb6-94ddf544a9c7.png)

I love to casually read through repos on my mobile, and having full screen in landscape would be awesome. It's a little strange to have the tab bar and heading showing, I suggest doing something like YouTube's touch to display. 

- Time out that causes headers and tabs to animate out while in landscape
- click (not scroll) to make them reappear
- repeat

**Comment by PranamBhat** [comment]
@swushi great input! Thank you!
